packages/muelu/test/scaling/recurMG.xml

```diff
@@ -0,0 +1,140 @@
+<ParameterList name="MueLu">
+
+  <Parameter name="verbosity" type="string" value="high"/>
+
+  <Parameter name="max levels" type="int" value="4"/>
+  <Parameter name="coarse: max size" type="int" value="150"/>
+  <Parameter        name="number of equations"                  type="int"      value="3"/>
+
+  <Parameter name="multigrid algorithm" type="string" value="sa"/>
+  <Parameter name="sa: damping factor"  type="double" value="1.33333"/>
+
+   <!-- THE BIG DATA DUMP -->
+    <ParameterList    name="export data">
+       <Parameter  name="P"                             type="string"   value="{1234}"/>
+       <Parameter  name="A"                             type="string"   value="{12345}"/>
+       <Parameter  name="Coordinates"                   type="string"   value="{30}"/>
+    </ParameterList>
```

Comment:
Does this even do anything?

Reply:
I suppose I should remove it.  This is my trick to "do nothing" when I want to keep something around ... and am too lazy to comment it out.  I'll get rid of it.